### ISSUE TITLE
Finetune result

### ISSUE THREAD

**Issue by mna12478** [chuanqi305/MobileNet-SSD#82]
When I finetune the model on mscoco 2017, I load the weights and trained for about 20000 epoches. When I test on the test files in VOC0712, the loss is 5.07, and the detection_eval is only 0.0013. So I wonder why this happens. Can anyone help me? By the way, should I change the train.protxt file? I noticed that someone proposed to delete the '_new' in 'conv11_mbox_conf_new', 'conv13_mbox_conf_new','conv14_2_mbox_conf_new'.
